lib/lightning/projects/provisioner.ex

```diff
@@ -78,6 +80,61 @@ defmodule Lightning.Projects.Provisioner do
     |> maybe_add_project_credentials(user_or_repo_connection)
   end
 
+  defp audit_workflows(project_changeset, user_or_repo_connection) do
+    project_changeset
+    |> get_assoc(:workflows)
+    |> Enum.reduce(
+      Multi.new(),
```

Comment:
Being already part of a transaction it wasn't necessary to be on a Multi. Nonetheless, I see little benefit from refactoring it to a `reduce_while` for now.